### ISSUE TITLE
Write the expansion when only detecting the abbreviations

### ISSUE THREAD

**Issue by donganzh-zz** [azu/textlint-rule-abbr-within-parentheses#1]
Hi, @azu, Here are some new features that would be nice to have.

Example situation:
`This is a text editing tool from OSS.`

When users are reading the document, they might not understand what `OSS` stands for. It is beneficial to prompt a warning message to the users to add the full expansion of those abbreviations so that it becomes the correct format of this rule. 

Furthermore, it would be better to have an exception list for those commonly-used abbreviations to not be expanded, such as `HTTP, URL, DIY, FBI`, etc. It should be customized by each user so we can put our own excluding abbreviations in a configuration file. Thank you again for this tool!

**Comment by azu** [comment]
Yeah, We need to support  allow options like https://github.com/textlint-rule/textlint-rule-en-capitalization#options

**Comment by donganzh-zz** [comment]
Here are some options. I think we need to enforce that acronyms should always exist with its expansion, and put those exclusion acronyms in the options list.
```{
    "rules": {
        "abbr-within-parentheses": {
             // allow lower-case words such as 'SaaS' in acronyms
             "allowLower": true,
             // allow digits such as 'B2B' in acronyms
             "allowDigits": true,
             // allow special character(-,_,/) such as 'TCP/IP' in acronyms
             "allowSpecial": true,
             // allow common acronyms that do not need to be expanded
             "commonAcronym": []
         }
    }
}

**Comment by amimas** [comment]
I think some of the suggestions are good; not sure about `allowDigits` option though.

Anyways, let's open separate issue for tracking each of the configuration options.
